awx/api/serializers.py

```diff
@@ -713,14 +714,17 @@ class UnifiedJobSerializer(BaseSerializer):
         read_only=True
     )
 
+    launched_by = serializers.SerializerMethodField(help_text=_('Object that launched the current job'))
+    ancestor_job = serializers.SerializerMethodField(help_text=_('The parent workflow that started the job, if not then ancestor_job returns self '))
```

Comment:
Does this _need_ to be exposed to the API, or is it just a function that's used internally by `launched_by`?

cc @marshmalien @wenottingham

Reply:
@ryanpetrello it does.  Per our conversation with @marshmalien a while back, the UI route built up to the `launched_by` object is relative to the job that it is referenced by, the `ancestor_job`.

Reply:
Gotcha 👍

Reply:
With the summary field added, we shouldn't need this `SerializerMethodField` anymore.

Reply:
I have removed it.